### DISCUSSION TITLE
failed to sent file_CAPSTONE BIKE SHARING API

### DISCUSSION THREAD

**Discussion by NiaMiz1** [teamalgoritma/community#830] [Q&A] [answered]
saya gagal mengirimkan satu file yang besarnya melebihi kapasitas yaitu 25MB

![gagal kirim file](https://github.com/teamalgoritma/community/assets/151926312/383f9630-4abc-4f6c-a45b-653471c2fef7)

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @NiaMiz1 ,

Untuk upload ke github cukup `app.py `dan `Bikeshare API.ipynb` beserta semua yg ada di dalam folder `data/`. Untuk file databasenya tidak diupload juga tidak apa-apa berhubung sizenya sangat besar Bu.

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.
